samples/sample-app-storage/build.gradle

```diff
@@ -45,15 +46,13 @@ dependencies {
     implementation "com.google.dagger:dagger-android-support:2.20"
     //noinspection GradleDynamicVersion
     implementation "com.microsoft.identity.client:msal:0.3+"
-    implementation "com.fasterxml.jackson.core:jackson-databind:$jacksonDatabindVersion"
-    implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-xml:$jacksonDataFormatXmlVersion"
-    implementation "com.jakewharton.threetenabp:threetenabp:$threeTenAbpVersion"
-    implementation "com.squareup.okhttp3:okhttp:$okHttpVersion"
-    implementation "com.squareup.retrofit2:retrofit:$retrofitVersion"
-    implementation "javax.xml.stream:stax-api:$staxApiVersion" // https://stackoverflow.com/a/47371517/1473510
-    // Unit testing
+    implementation "com.fasterxml.jackson.core:jackson-databind:2.9.9"
```

Comment:
On the same vein as the buildToolsVersion question, is the reason we hard-coding dependencies here is because we want it to be a complete separate package?